docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2858,6 +2861,26 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+
+  Discriminator:
+    type: object
+    description: |
+      Event field parameters to differentiate consumption of events published on per-consumer level.
+      The discriminator can be used to specify a field to be used from the events published to this event type.
+      This field can then be used to implement logic using a plugin to whether an event would be provided
+      in a consumer's stream (`EventsStreamBatch` or `SubscriptionEventStreamBatch`).
+      Once created, the discrimator definition for an event-type can't be updated or removed.
+    properties:
+      path:
+        type: string
+        description: |
+          Path of the field in the event that contains the identifiers of the entities that can access the event.
+      type:
+        type: string
+        description: |
+          Type of the field specified by the path the contains the identifiers of entities.
```

Comment:
Why is the type needed here? Can't this be derived from the schema itself?

Reply:
Right now, for existing event types, we have some users wanting to implement per-event authorization on consumer end without changing the event type schema or introducing a field (required or not) for all publishers. So, as a result, we accept the type of the field too.

Reply:
Ah, then this should mention that the field does not need to be to exist in the event schema itself (though then it's not quite clear to me on what happens if the field does not exist, or has a different type).

Reply:
Apologies, I misunderstood the `type` of the field. After clearing this up, I've updated this in `f22915a7f`. 
```
path: Path to a string field in the event that contains the a classifier, which can be used by authorization plugin (together with definitions in the authorization section) to define who can read the event. The field may not be present in the schema definition for the event type. In case the field is present, it will be available for reading for all authorized consumers.
type: Indicator of the type of the data (classifier) the field contains (eg: `retailer_id` or `application_name`).
```

I've updated the documentation to update what happens if the field is null/absent in `9158466b7`. Thanks for this pointing out. :)